### DISCUSSION TITLE
DNS Rebinding

### DISCUSSION THREAD

**Discussion by larivierec** [0xERR0R/blocky#702] [Q&A] [answered]
I'm not sure if this is the right place for this but does blocky support DNS rebinding?
For exemple usage with plex.

I know that It's supposed on unbound DNS which I use as a secondary DNS but was wondering if it worked with blocky.

**Comment by kwitsch** [comment]
Blocky supports custom DNS entries, conditional upstream resolvers and domain rewrite rules.

What would be your use case?
Maybe I can help with the configuration.

**Reply by larivierec** [reply]
Hello again :)

The section it's referring to is this url https://support.plex.tv/articles/206225077-how-to-use-secure-server-connections/ "dns rebinding protection"

I'm not really sure if it's just an option or specifically forcing an IP, but with unbound and dnsmasq this seems possible.

Wish I could give you better information

**Reply by 0xERR0R** [reply, marked as answer]
I think you mean "DNS rebinding protection". Basic idea: some routers or even ISPs block DNS responses which point to a local network (for example if google.com will point to 192.168.x.x). Blocky doesn't apply any rules automatically to block DNS queries, so the default behavior - no rebinding protection.

**Reply by larivierec** [reply]
It's all good
Turns out my DNS was just fine

Blocky doing what it's doing 😄
Sorry for the bother, I even tried to wire shark capture... turns out it was an ingress redirect.

Do you want me to delete the post?
